tck/features/SkipLimitAcceptance.feature

```diff
@@ -30,24 +30,24 @@
 
 Feature: SkipLimitAcceptanceTest
 
-  Background:
-    Given any graph
-
   Scenario: SKIP with an expression that depends on variables should fail
+    Given any graph
     When executing query:
       """
       MATCH (n) RETURN n SKIP n.count
       """
     Then a SyntaxError should be raised at compile time: NonConstantExpression
 
   Scenario: LIMIT with an expression that depends on variables should fail
+    Given any graph
     When executing query:
       """
       MATCH (n) RETURN n LIMIT n.count
       """
     Then a SyntaxError should be raised at compile time: NonConstantExpression
 
   Scenario: SKIP with an expression that does not depend on variables
+    Given an empty graph
```

Comment:
I don't see why this can't be `any graph`?

Reply:
Since this one only returns whether the count is greater than 0, I agree.